### PR TITLE
PRIME-2679 Email Template Update

### PR DESCRIPTION
**Pull request by bergomi02**
https://pr-2462.pharmanetenrolment.gov.bc.ca/admin

### CODE REVIEW

**Comment by neophyte57** [comment]
Try to update database in PR environment

**Comment by neophyte57** [comment]
Something strange re: "Last Modified" date on the email template cards.  In the PR environment, only the two updated templates have a date other than "16 Sep 2019".  In `dev`, all have "Last Modified" of "16 Sep 2019".  Whereas `test` and my local environment is actually as expected with a mix of "Last Modified" dates.  Why is that?

File: prime-dotnet-webapi/Migrations/20240318232909_EmailTemplateUpdate.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Prime.Migrations
+{
+    public partial class EmailTemplateUpdate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "PharmanetTransactionLogTemp");
```
Comment (neophyte57) [inline prime-dotnet-webapi/Migrations/20240318232909_EmailTemplateUpdate.cs:12]: This snuck in?

Comment (bergomi02) [reply prime-dotnet-webapi/Migrations/20240318232909_EmailTemplateUpdate.cs]: Not sure why. I re-generated the migration and it did the same. For now, I removed the code manually for now.

File: prime-dotnet-webapi/Migrations/ApiDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -12059,60 +12059,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("PharmanetTransactionLog");
                 });
 
-            modelBuilder.Entity("Prime.Models.PharmanetTransactionLogTemp", b =>
```
Comment (neophyte57) [inline prime-dotnet-webapi/Migrations/ApiDbContextModelSnapshot.cs:12062]: Shouldn't `PharmanetTransactionLogTemp` remain in `ApiDbContextModelSnapshot`?

Comment (bergomi02) [reply prime-dotnet-webapi/Migrations/ApiDbContextModelSnapshot.cs]: Good catch. Not sure how it got removed from the snapshot. I added it back manually. It should work fine now.

**Review by neophyte57 — APPROVED** [review]
LGTM

**Pull request merged**